pgd_core/templates/profile.html

```diff
@@ -5,6 +5,31 @@
 
 {% block content %}
 	<head> Welcome, {{full_name}} </head>
+
+	<div style="position:fixed; top:70px; right:90px; width:200px;">
+		<form method="post">{% csrf_token %}
+
+			{{ form.query.label }} <br>
+			{{ form.query}} <br>
+			{% for choice in form.search_type %}
+	  			{{ choice.choice_label }}
+	  			<span class="radio"> {{ choice.tag }} </span> <br>
+			{% endfor %}
+
+		</form>
+	</div> 
+
+<style>
```

Comment:
Lines 22 through 31 used to be down near line 71.  Why was this code moved upwards?

Also, the indentation has moved out flush with the left margin, instead of indented one tab stop or so as it used to be.  Why?